trackpy/linking.py

```diff
@@ -1423,7 +1418,8 @@ def numba_link(s_sn, dest_size, search_radius, max_size=30, diag=False):
     dest_results = [dcands[i] if i >= 0 else None for i in best_assignments]
     return source_results, dest_results
 
-@try_numba_autojit
+import numba
+@numba.autojit(nopython=True)
```

Comment:
Was this accidentally committed? This would definitely break without numba installed.